src/GitHub.VisualStudio/UI/Settings/OptionsControl.xaml

```diff
@@ -81,6 +81,22 @@
       </TextBlock>
       </DockPanel>
     </GroupBox>
+    <GroupBox
+            Margin="8,0,8,0"
+            VerticalAlignment="Top"
+            BorderBrush="LightGray"
```

Comment:
Not sure why this `<WrapPanel>` is necessary seeing as there is only one child (a `WrapPanel` with one child won't do anything as far as I understand) but I see the other sections are the same so /shrug.

Reply:
It does not seem to have an effect on this element.